### ISSUE TITLE
Cant connect to server through jellyfin media player

### ISSUE THREAD

**Issue by WoxWik** [AdrienPoupa/docker-compose-nas#19]
I did the standard install and can access jellyfin through https://omv.local/jellyfin in my webbrowser, but I get a connection failure when pasting that same url in jellyfin media player. I'm using the TRAEFIK DEFAULT CERT since I didn't want to sign up for cloudflare. Any tips?

**Comment by AdrienPoupa** [comment]
Hello,

> I'm using the TRAEFIK DEFAULT CERT since I didn't want to sign up for cloudflare

This is probably why. Most likely Jellyfin media player is not accepting self-signed certificates by default. You may want to try this: https://github.com/jellyfin/jellyfin-media-player/issues/48#issuecomment-822902833

That being said, you can use Let's Encrypt without CloudFlare. Just select the DNS01 provider of your registrar (see documentation, the `DNS_CHALLENGE_PROVIDER` variable and the list here: https://doc.traefik.io/traefik/https/acme/#providers) and you should be good to go.

**Comment by WoxWik** [comment]
Hi Adrien, thanks for the speedy response. I dont think letsencrypt will work with my .local domain since these aren't regulated but correct me if I'm wrong. I disabled ssl checking as mentioned in  your link, and fixed my local dns.

Apparently web browsers can automatically resolve through mdns which is what my server was broadcasting on it's own. This is why it was working through my browser and not the jellyfin media player application.

I resolved the issue by adding my servers ip to the hosts file.
